semantic-web-technology-introduction.markdown

```diff
@@ -12,26 +12,25 @@ In general a growing number of people produce in their activities, both professi
 This heterogeneity has several causes.  
 Humans have developed more than 7800 **natural languages**, each of them with more or less own interpretation and description models of the human observation of reality. Even when different languages belong to the same group, a 1-1 translation is often impossible.  
 Within a language, the way people describe things with a certain vocabulary or **terminology** or jargon differs along education, science, occupation, industry and other **domains**. But also intra-disciplinary terms not always bear exactly the same meaning.  
-On the technological side, a lot of data are **little or not structured** (free text). One can secondarily structure data through parsing the text, but unless the accuracy of text recognition is 100%, a secondary data error is introduced. Creating possibilities to up-front input sufficiently structured data might be an easier, cheaper and more reliable approach.  
-Last but not least, even when data input is structured in databases (e.g. in XML or SQL), and even when 2 databases contain very similar information, their schemas or **data models**, though capturing consented domain knowledge (as found in literature, or in tacit knowledge in the heads of domain specialists) are often **localized** through natural language and personal representations such as abbreviations and cryptic naming, not fully described. In other words they are not always stating meaning explicitly.  
+On the technological side, a lot of data are **little or not structured** (free text). One can structure data through parsing the text, but unless the accuracy of text recognition is 100%, data errors are introduced. Creating possibilities to up-front input sufficiently structured data might be an easier, cheaper and more reliable approach. But even if data input is structured in databases (e.g. in XML or SQL), their schemas or **data models**, though capturing consented domain knowledge (as found in literature, or in tacit knowledge in the heads of domain specialists) are often **localized**# through natural language and personal representations such as abbreviations and cryptic naming and not fully described. In other words they are not always stating meaning explicitly.  
```

Comment:
#: Is localized the right word?

Reply:
localized -?> personalized: in the sense of not understandable for a broader group